compute/instances.go

```diff
@@ -961,17 +984,14 @@ func (api *_Instance) toNative() (*Instance, error) {
 
 // toTags() injects its state information into a Tags map suitable for use to
 // submit an API call to the vmapi machine endpoint
-func (mcns *InstanceCNS) toTags(m map[string]interface{}) {
-	if mcns.Disable != nil {
-		s := fmt.Sprintf("%t", mcns.Disable)
-		m[CNSTagDisable] = &s
+func (cns *InstanceCNS) toTags(m map[string]interface{}) {
+	if cns.Disable {
+		m[CNSTagDisable] = cns.Disable
 	}
-
-	if mcns.ReversePTR != nil {
-		m[CNSTagReversePTR] = &mcns.ReversePTR
+	if cns.ReversePTR != "" {
+		m[CNSTagReversePTR] = cns.ReversePTR
 	}
-
-	if len(mcns.Services) > 0 {
-		m[CNSTagServices] = strings.Join(mcns.Services, ",")
+	if len(cns.Services) > 0 {
+		m[CNSTagServices] = strings.Join(cns.Services, ",")
```

Comment:
In order to get a stable comparison, in both the *Read call and here, do we need to sort `cns.Services`?  I don't know but seem to recall there wasn't a sorting guarantee here (I also don't think we should depend on it if there is one currently).